server/public/model/job.go

```diff
@@ -44,6 +44,7 @@ const (
 	JobTypeExportUsersToCSV              = "export_users_to_csv"
 	JobTypeDeleteDmsPreferencesMigration = "delete_dms_preferences_migration"
 	JobTypeMobileSessionMetadata         = "mobile_session_metadata"
+	JobTypeUserDeletion                  = "permanent_delete_user"
```

Comment:
nit: Constant name and value are a bit inconsistent. I'd expect the const to be `JobTypePermanentDeleteUser`.